Build/run.sh

```diff
@@ -64,21 +64,29 @@ if [ ! -d "${OUTDIR}" ];
 then
 	mkdir ${OUTDIR}
 fi
-chmod 777 ${OUTDIR} ${IGNORE_OUTPUT}
+chmod +w ${OUTDIR} ${IGNORE_OUTPUT}
+ret=$?
+if [ $ret -ne 0 ];
+then
+	echo "Unable to obtain full acess  permissions to ${OUTDIR} and its sub directories, edit the permissions of ${OUTDIR} accordingly! exit"
+	exit -1
+fi
 if [ -z "${CPATH}" ]; 
 then
   /usr/bin/docker run --privileged ${USE_TTY} --rm -e NO=${NO} -e SDK=${SDK} -e DLOAD=${DLOAD} -v "${PWD}/${OUTDIR}":/home/yocto/rzv_vlp_v3.0.0/out --name ${CONTNAME} ${CONTNAME}
 else
 	#Create CPATH sub directories if they do not exist
 	if [ ! -d "${CPATH}/downloads" ];
 	then
-		mkdir ${CPATH}/downloads
+		mkdir -p ${CPATH}/downloads
+		mkdir -p ${CPATH}/sstate-cache/${MPU}
```

Comment:
if you are using `-p` then you don't need to use an if condition to check for existence. It wouldn't throw an error if it exists.